homeassistant/components/yeelight/light.py

```diff
@@ -414,18 +415,23 @@ class YeelightGenericLight(YeelightEntity, LightEntity):
     )
     _attr_should_poll = False
 
-    def __init__(self, device, entry, custom_effects=None):
+    def __init__(
+        self,
+        device: YeelightDevice,
+        entry: ConfigEntry,
+        custom_effects: dict[str, dict[str, Any]] | None = None,
+    ) -> None:
         """Initialize the Yeelight light."""
         super().__init__(device, entry)
 
         self.config = device.config
 
-        self._color_temp = None
+        self._color_temp: int | None = None
```

Comment:
This is the default

```suggestion
```

Reply:
This is not using the attribute so it still needs defining.

Reply:
Oh right 🤦 

Misread on my end, sorry about that 👍